ansible/module_utils/graph_utils.py

```diff
@@ -0,0 +1,481 @@
+import csv
```

Comment:
The "module_utils" folder is for code supposed to be shared by multiple ansible modules. Wondering why move the LabGraph class to this folder. Currently it is only used by the conn_graph_facts.py module, correct?

Reply:
Not completely true. It is used by devutils too, so I figured it is kinda common and will be great to have it placed in a common area, like "sdk"-ish.